### DISCUSSION TITLE
Mesoscale eddy closure in Veros

### DISCUSSION THREAD

**Discussion by mrozowskamartaa** [team-ocean/veros#680] [Q&A]
The pyOM2.0 documentation [1] indicates that the mesoscale eddy closure in Veros should use the Eden & Greatbatch (2008) [2] parameterization. However, the code in veros/core/eke.py indicates that the eddy kinetic energy equation is of a different form than the one given by E&G (2008). Is there another publication or documentation of the version of E&G closure used in Veros?

[1] [pyOM2_VerosCore_Eden.pdf](https://github.com/user-attachments/files/18108618/pyOM2_VerosCore_Eden.pdf)
[2] [EdenGreatbatch2007.pdf](https://github.com/user-attachments/files/18108611/EdenGreatbatch2007.pdf)

Specifically, it seems that the vertical diffusion of EKE is computed using a different term in the E&G EKE budget and in the integrate_eke_kernel() function in Veros. The latter includes vertical eddy diffusivity coefficients kappaM and alpha_eke, both absent from the E&G vertical diffusion term.

**Comment by dionhaefner** [comment]
This is taken verbatim from PyOM, so I suggest you ask there. Unfortunately there's no other documentation that I know of.

**Reply by mrozowskamartaa** [reply]
Thx!
